api/src/main/java/jakarta/data/page/Page.java

```diff
@@ -146,8 +146,18 @@ default Stream<T> stream() {
      * Returns the total number of elements across all pages of query results, if the
      * {@link #pageRequest()} specified that {@linkplain PageRequest#requestTotal the
      * total should be retrieved from the database}.
+     * {@apiNote Not all NoSQL databases support counting the total
+     * number of elements. This operation is not supported for Key-Value and Wide-Column databases.
+     * For Graph and Document databases, support for this operation may vary depending on the provider.
+     * If the database does not support retrieving the total number of elements, calling this method will
+     * result in an {@link UnsupportedOperationException}.</p>
+     *}
```

Comment:
`{@apiNote ...}` in curly braces didn't seem to work when I tried it. Should it instead be formatted like `@param` or `@return` ?

```suggestion
     * @apiNote Not all NoSQL databases support counting the total number of elements.
     *          This operation is not supported for Key-Value and Wide-Column databases.
     *          For Graph and Document databases, support for this operation may vary
     *          depending on the provider. If the database does not support retrieving the
     *          total number of elements, calling this method will result in an
     *          {@link UnsupportedOperationException}.
```